test/any/web_apis/api_importer-test.es6.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * @license
+ * Copyright 2017 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the 'License');
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an 'AS IS' BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+describe('ApiImporter', function() {
+  let apiImporter = org.chromium.apis.web.ApiImporter.create();
+  let mlang = foam.mlang.ExpressionsSingleton.create();
+  let webCatalog = {
+    'Window': [
+      'Function',
+      'property',
+    ],
+    'Function': [
+      'arguments',
+      'caller',
+    ],
+  };
+
+  apiImporter.import('Chrome', '56.0.2924.87',
+    'OSX', '10.12.2', webCatalog);
+  it('correctly imports browserAPIs to MDAO', function(done) {
+    let interfaces = [
+      {
+        interfaceName: 'Window',
+        apiName: 'Function',
+      },
+      {
+        interfaceName: 'Window',
+        apiName: 'property',
+      },
+      {
+        interfaceName: 'Function',
+        apiName: 'arguments',
+      },
+      {
+        interfaceName: 'Function',
+        apiName: 'caller',
+      },
+    ];
+    let promises = interfaces.map((intface) => {
```

Comment:
This is probably overkill for this test. We're not testing the soundness of the DAO query API, we're checking whether the DAO contains what we want it to contain, _and nothing more_.I was expecting something like:

```
let dao = apiImporter.browserAPIs;
let promises = [ dao.find(...), dao.find(...), ... ];
Promise.all(promises).then(function(results) {
  /* Make sure all finds are non-null */
  return dao.select(mlang.COUNT());
}).then(function(countSink) {
  /* Check that count matches promises.length */
});
```

This method has the weakness that if we mistakenly `find()` the same value twice, then our `COUNT()` check is inconsistent. However, that should be easy to spot, and in the absence of such an error, we check that the DAO doesn't contain more than it should.